documentation/modules/configuring/con-config-mirrormaker2-replication.adoc

```diff
@@ -15,6 +15,10 @@ MirrorMaker 2.0 uses:
 MirrorMaker 2.0 is based on the Kafka Connect framework, _connectors_ managing the transfer of data between clusters.
 A MirrorMaker 2.0 `MirrorSourceConnector` replicates topics from a source cluster to a target cluster.
 
+By default, a check for new topics in the source cluster is made every 10 minutes.
+You can change the frequency by adding `refresh.topics.interval.seconds` to the source connector configuration of the `KafkaMirrorMaker2` resource.
+However, increasing the frequency of the operation might affect overall performance.
+
```

Comment:
I have limited knowledge of MM2, but the new paragraph seems to interrupt the flow of the original introduction (which is great). It talks about the `KafkaMirrorMaker2` resource very early on.

Perhaps it could be positioned after line 25, or even in the "Cluster configuration" section.

Reply:
Fair point. I've moved to after the image